README.md

```diff
@@ -148,6 +152,7 @@ teleop:                        # Parameters for the Teleoperate mode.
   increment_step: 0.1          # Step for increasing the velocity increment.
   cmd_vel_topic: cmd_vel       # Topic on which to publish the velocity commands.
   publish_cmd_vel_when_idle: true # If true keep publishing 0 velocities, only publish once otherwise
+tf_frames_service_name: /tf_service/tf2_frames
```

Comment:
Initially i was pretty sure this service available always: https://github.com/ros-visualization/rqt_tf_tree/blob/c7e314dd835f7cc238e1dccb259bc709417b6370/src/rqt_tf_tree/tf_tree.py#L148. But then when I launched turtlebot demo, you are right, the service wasn't there, forgot to add a TODO.

But apparently it's only shortlived when we create tf2_buffer and with limited scope?

If that's the case, probably adding to README that we need to run tf_service is ok, since I don't know the better solution for this. Rust tf implementation is very stripped, at first I tried to do that without relying on a service, but there are simply too many fixes needed to be done there.

Reply:
I've added explanation in README above about the service. I don't like that you need to have something else running in order for this widget to work to be honest, I was sure the service will be there always... If I will have time, maybe I will work on rust tf implementation, so it can incorporate tf2_frames service there.

As for `fix indentation` comment - I am really not sure about what you are talking about? YAML seems to be correct, frames service name param is not nested anywhere

Reply:
True about the indentation. I looked wrong at the diff.

Reply:
not a big fan of requiring a third party service running for the feature to work.  We are using a fork of rustros_tf anyways. Maybe add the feature there? https://github.com/maximaerz/rustros_tf i am sure the maintainer will cooperate. @MaxiMaerz ;)

Reply:
Agree, probably dependency on 3rd party service makes feature useless for a lot of people. 

I've looked up what's required for rustros_tf, tl;dr I need to rewrite all frame buffering logic there, it's doable but quite some work, so no promises on timelines. We can close this pr for now or convert to draft if that will be better?